saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -34,11 +42,13 @@ setup_ccmv2() {
   INVERTING_PROXY_URL="$CCM_V2_INVERTING_PROXY_HOST"
 
   if [[ "$IS_CCM_V2_JUMPGATE_ENABLED" == "true" && "$IS_FREEIPA" == "true" && ! -z $CCM_V2_AGENT_ACCESS_KEY_ID ]]; then
-    ACCESS_KEY_ID="$CCM_V2_AGENT_ACCESS_KEY_ID"
 
     if [[ -z $CCM_V2_AGENT_HMAC_KEY ]]; then
       # legacy without HMAC check, AES-128
       ACCESS_KEY="$(echo ${CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY} | openssl enc -aes-128-cbc -d -A -a -K ${CCM_V2_AGENT_KEY_HEX} -iv ${LEGACY_IV})"
```

Comment:
This will reveal the result of the command substitution. Example:

```
❯ echo -e '#!/bin/bash\nset -x\nACCESS_KEY="$(date)"\necho "ACCESS_KEY=$ACCESS_KEY" > sensitive.txt\ncat sensitive.txt\nrm sensitive.txt\n' > sensitive.sh; chmod +x sensitive.sh; ./sensitive.sh ; rm sensitive.sh
++ date
+ ACCESS_KEY='Wed 04 Dec 2024 12:10:28 PM CET'
+ echo 'ACCESS_KEY=Wed 04 Dec 2024 12:10:28 PM CET'
+ cat sensitive.txt
ACCESS_KEY=Wed 04 Dec 2024 12:10:28 PM CET
+ rm sensitive.txt
```